testsuite/cluster-test/src/cluster_builder.rs

```diff
@@ -163,13 +176,27 @@ impl ClusterBuilder {
                 .collect();
             lsrs.append(&mut lsr_instances);
         }
+
+        let validator_nodes = try_join_all((0..num_validators).map(|i| async move {
+            let pod_name = format!("val-{}", i);
```

Comment:
Do you mind moving this format! to Instance as some static method (like `Instance::validator_pod_name(index)`), and modify Instance::pod_name so it calls to this method. Basically, I want to make sure that `format!("val-{}")` happens exactly once(at least in rust code) so that if we change it, there is only one place to update

Reply:
sure.